qa/pull-tester/rpc-tests.py

```diff
@@ -115,6 +115,8 @@
 ] ]
 
 testScriptsExt = [ RpcTest(t) for t in [
+    'txPerf',
+    'excessive --extended',
```

Comment:
Its mostly more iterations.  In one case I skipped a test in the non-extended version since the test takes a long time and can't be shortened.  It has to create 1MB+ of transactions and then create and sign a large tx.  This takes time.